docs/sources/setup-grafana/configure-grafana/feature-toggles/index.md

```diff
@@ -71,7 +71,6 @@ Alpha features might be changed or removed without prior notice.
 | `storage`                          | Configurable storage for dashboards, datasources, and resources                                                                                                                                     |
 | `exploreMixedDatasource`           | Enable mixed datasource in Explore                                                                                                                                                                  |
 | `newTraceView`                     | Shows the new trace view design                                                                                                                                                                     |
-| `correlations`                     | Correlations page                                                                                                                                                                                   |
```

Comment:
This change is not related and will be removed when https://github.com/grafana/grafana/pull/66499 is merged.